nimby-vs-yimby/README.md

```diff
@@ -0,0 +1,303 @@
+#fig.show()
+fig.write_image("per_capita_by_county.png")
+Image(filename="per_capita_by_county.png")
+```
+
+
+
+
+    
+![png](output_6_0.png)
+    
+
+
+
+
+```python
+fig = px.bar(df_counts_by_county, x='county_state', y='per_capita_stdevs_from_mean', title="New real estate sales per capita (stdevs. from mean)", labels=labels)
+#fig.show()
+fig.write_image("per_capita_stdevs_by_county.png")
+Image(filename="per_capita_stdevs_by_county.png")
+```
+
+
+
+
+    
+![png](output_7_0.png)
+    
+
+
+
+We can see that the spread is quite severe. Some counties have over two standard deviations more real estate developments more than the mean value (per capita). Some other counties lag behind by having their per capita number lower than one standard deviation below the mean. Let us look into some outliers on both extremes to try getting some ideas for explanation.
+
+What are top 10 counties we know of with highest per capita number of new real estate projects?
+
+
+```python
+df_best_per_capita = df_counts_by_county.sort_values('per_capita', ascending=False).head(10)
+
+fig = px.bar(df_best_per_capita, x="county_state", y="per_capita", labels=labels)
+#fig.show()
+fig.write_image("top10_per_capita_by_county.png")
+Image(filename="top10_per_capita_by_county.png")
+```
+
+
+
+
+    
+![png](output_9_0.png)
+    
+
+
+
+* Cape May county, NJ lays on souther New Jersey and is quite attractive to tourists, which also makes it attractive to real estate investors. This county is somewhat urbanised by coastal beach towns and has a moderate population density - largest in the top 10 list, but significantly lower than that in San Francisco.
+* Grand County, CO is mostly rural place with low population density. The same applies to all entries on top 10 list except Cape May, NJ. It's probably not significant that these locations have high per capita amount of real estate developments, given that populations are small and per area amounts of new projects are below average. Unsurprisingly, lack of population means lack of NIMBYs.
+
+What are the 10 counties with lowest per capita amounts of new real estate projects?
+
+
+```python
+df_worst_per_capita = df_counts_by_county.sort_values('per_capita', ascending=True).head(10)
+fig = px.bar(df_worst_per_capita, x="county_state", y="per_capita", labels=labels)
+#fig.show()
+fig.write_image("worst10_per_capita_by_county.png")
+Image(filename="worst10_per_capita_by_county.png")
+```
+
+
+
+
+    
+![png](output_11_0.png)
+    
+
+
+
+* Arlington, VA had only one sales record scraped during the bounty, so it's safe to say this is statistical anomaly.
+* Most (but not all) of these areas are more urbanised with population density exceeding 200 people per square mile.
+* Indian river county is on the eastern coast of Florida that is home to some fairly wealthy beach towns. 
+* Polk county, FL is fairly populous urban area with central Florida with know resident opposition to new real estate projects.
```

Comment:
```suggestion
* Polk county, FL is fairly populous urban area in central Florida with known resident opposition to new real estate projects.
```

Link?

Reply:
Link added. Markdown will be regenerated. The same applies to all grammar/spelling fixes.